### ISSUE TITLE
TypeError: style.toFontString is not a function

### ISSUE THREAD

**Issue by patchanopjob** [Mwni/pixi-text-input#45]
I use Pixijs8

![Screenshot 2567-09-19 at 09 58 07](https://github.com/user-attachments/assets/88b68278-3ac7-46ea-92af-6a2c4dab64c0)
![Screenshot 2567-09-19 at 09 58 31](https://github.com/user-attachments/assets/f5659b0d-5a5e-412b-b603-95c04bd67bcc)

**Comment by Mwni** [comment]
You're right, it's broken for v7 and higher. I will probably fix this some time later today.

**Comment by amitp2025** [comment]
Hi @Mwni,

Is this issue fixed? I am still getting this issue when using pixijs v8.
